src/index.ts

```diff
@@ -1 +1,4 @@
-// Nothing here yet
+import { HttpServer, HttpServerOptions } from './http-server'
+import { HttpsServer, HttpsServerOptions } from './https-server'
+
+export { HttpServer, HttpServerOptions, HttpsServer, HttpsServerOptions }
```

Comment:
Maybe we want to package all of these http utils in a http namespace (and folder)?

Reply:
Good idea, I'm not sure how to do this nicely when we can't use subfolders, it makes the import a bit less nice if you have to do:

``` typescript
import { http } from '@connectedcars/test'
const { HttpServer } = http
```

Do you have a good alternative?

Reply:
I don't really have one. Bit annoying you can't do object destructering in the import statement. But I think it would make sense to move the files to an http folder and still export them in the default namespace in the index file.

Reply:
Could also be it was ts-lint that had an opinion using subpath imports, but I never really understood why it was bad

Reply:
I think the issue is that the interface becomes more ambiguous. We could also move completely to subpath imports so we don't have a main entrypoint but I'm not sure that's the best way to go.